test/python/test_operation.py

```diff
@@ -992,18 +992,55 @@ def test_Sqrt_gpu(self):
         np.testing.assert_array_almost_equal(tensor.to_numpy(result), XT, decimal=5)
         np.testing.assert_array_almost_equal(tensor.to_numpy(tensor.from_raw_tensor(dx)), DX, decimal=5)
 
+        
+    def test_transpose_cpu(self):
+        x = np.array([0.1,-1.0,0.4,4.0,-0.9,9.0]).reshape(3,2,1).astype(np.float32)
+        y = x.transpose(1,2,0)
+        dy = np.ones(y.shape, dtype = np.float32)
+        grad = np.ones((3,2,1), dtype = np.float32)
```

Comment:
use np.random.rand

Reply:
what is grad?

Reply:
I just changed my wrong code.
grad is dx and I used transpose for dy